erigon-lib/state/domain_test.go

```diff
@@ -2082,230 +1753,6 @@ func TestDomain_PruneProgress(t *testing.T) {
 	fmt.Printf("exitiig after %d iterations\n", i)
 }
 
-func TestDomain_Unwind(t *testing.T) {
-	db, d := testDbAndDomain(t, log.New())
-	defer d.Close()
```

Comment:
why remvoe this test? Domains still provide Unwind functionality, that was naive test to check correctness after several unwinds on same data

Reply:
Because it tests unwind to tx

Reply:
this unit allows to check if some regression made in one Domain without being affected by others

Reply:
current unwind unwinds all domains so it is obsolete